README.md

```diff
@@ -10,24 +10,40 @@
 # cromshell
  A script for submitting workflows to a cromwell server and monitoring / querying their results.
```

Comment:
I believe it's no longer a single script, but rather an CLI. No?

Reply:
True